apps/api/src/app/integrations/usecases/get-active-integration/get-active-integration.usecase.ts

```diff
@@ -35,6 +46,10 @@ export class GetActiveIntegrations {
       return [];
     }
 
+    if (!isMultiProviderConfigurationEnabled) {
+      return activeIntegrations;
+    }
```

Comment:
when the feature flag is false then the use-case should behave like it was before